test/tstDetailsTypeRequirements.cpp

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ * Copyright (c) 2017-2021 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <ArborX_LinearBVH.hpp>
+
+#include <Kokkos_Core.hpp>
+
+namespace Test
+{
+// NOTE only supporting Point and Box
+using FakePrimitive = ArborX::Point;
```

Comment:
Would you like me to rename to `PrimitivePointOrBox` or similar?

Reply:
Yes, thank you.